pyproject.toml

```diff
@@ -19,7 +19,7 @@ dynamic = [
 ]
 dependencies = [
   "httpx",
-  "pandas",
+  "pandas>=2",
```

Comment:
The old code (without `format="ISO8601"`) was broken and failing, hence had to add it. We now need to specify that as a minimum version because `format="ISO8601"` isn't supported pre version 2.

Reply:
Interesting. I'm wondering how that could silently break :thinking: Let's :crossed_fingers: that this doesn't introduce dependency conflicts for our users because pandas 2.0 is only 2 weeks old.